back/controllers/rangeSupervision.js

```diff
@@ -34,13 +39,56 @@ const controller = {
   },
 
   create: async function createSupervision(request, response) {
+    
+    try{
+      //fetches the supervisor id who is assgined to the range.
+      const vastaanottaja = await geetUserEmail(response.req.body.supervisor);
+      //knex part that should be done in models? returns the email based on the id fetched above:
```

Comment:
Variable names in English (vastaanottaja -> receiver)

Reply:
done